omnigan/trainer.py

```diff
@@ -767,9 +767,13 @@ def get_d_loss(self, multi_domain_batch, verbose=0):
         Returns:
             [type]: [description]
         """
+<<<<<<< Updated upstream
         zerotensor = torch.tensor(0).to(self.device)
+=======
+        zerotensor = torch.tensor(0.0).to(self.device)
+>>>>>>> Stashed changes
```

Comment:
@tianyu-z you have this git marker trailing. Remember to run tests often! They wouldn't have passed

Reply:
Thanks!